test-framework/sudo-compliance-tests/src/timestamp.rs

```diff
@@ -174,12 +192,12 @@ fn cached_credential_shared_with_target_user_that_is_self() -> Result<()> {
 
     for _ in 0..5 {
         if let Some(status) = child.try_wait()? {
-            assert!(status.success());
+            assert!(!status.success());
             return Ok(());
         }
 
         thread::sleep(Duration::from_secs(1));
     }
 
-    panic!("timed out")
```

Comment:
so with `Defaults use_pty` *both* ogsudo and sudo-rs are blocking forever for user input. I don't think we have seen that before in any other tests; not providing input made `output` (eventually) return with `success = false` and `stderr = "authentication failed"`. perhaps we didn't test the 'no password provided scenario' with `Env.tty(true)`. 

in any case, I'm not happy with the polling approach used here because the `try_wait` never returns `Ok` so this is effectively doing `spawn; sleep(5); Ok(())` and not asserting anything. that being said, I don't have a suggestion on how to unstuck this :-/

maybe we could remove this test for now and open an issue to track adding a test (that actually asserts something) for this scenario

Reply:
Of course the easy road here is to use `-n` now that we support that. :)